google-cloud-clients/google-cloud-datastore/src/main/java/com/google/cloud/datastore/BaseEntity.java

```diff
@@ -636,6 +640,11 @@ public Blob getBlob(String name) {
     return properties;
   }
 
+  /** Returns the entity size. */
```

Comment:
"Returns the number of bytes in the serialized protobuf."

Is this definitely a fixed and reproducible value for a given protobuf?

Reply:
Yes,Once build it would be fixed number.